packages/react-core/src/components/Form/Form.tsx

```diff
@@ -9,24 +9,34 @@ export interface FormProps extends React.HTMLProps<HTMLFormElement> {
   className?: string;
   /** Sets the Form to horizontal. */
   isHorizontal?: boolean;
-  /** Flag to limit the max-width to 500px. */
+  /** Limits the max-width of the form. */
   isWidthLimited?: boolean;
+  /** Sets a custom max-width when using isWidthLimited. */
+  maxWidth?: string;
 }
 
 export const Form: React.FunctionComponent<FormProps> = ({
   children = null,
   className = '',
   isHorizontal = false,
   isWidthLimited = false,
+  maxWidth = '',
   ...props
 }: FormProps) => (
   <form
     noValidate
+    {...(maxWidth && {
+      style: {
+        '--pf-c-form--m-limit-width--MaxWidth': maxWidth,
+        ...props.style
+      } as React.CSSProperties
+    })}
     {...props}
     className={css(
       styles.form,
       isHorizontal && styles.modifiers.horizontal,
       isWidthLimited && styles.modifiers.limitWidth,
+      maxWidth && styles.modifiers.limitWidth,
```

Comment:
I *think* we would want to apply limit width like `(isWidthLimited || maxWidth) && styles.modifiers.limitWidth`, as applying it in the current manner would cause the class name to be applied twice if both `isWidthLimited` and `maxWidth` are applied.

Reply:
doh! I actually meant to remove this line before committing 😅